cognite_toolkit/_cdf_tk/tk_warnings/base.py

```diff
@@ -20,23 +20,23 @@ class SeverityLevel(Enum):
     MEDIUM = "yellow"
     LOW = "green"
 
-
-class SeverityFormat:
-    @staticmethod
-    def get_rich_severity_format(severity: SeverityLevel, *messages: str) -> str:
-        if severity == SeverityLevel.ERROR:
-            return f"[bold red]ERROR [{severity.name}]:[/] {' '.join(messages)}"
```

Comment:
I removed this class, moved the prefix part to the enum class and then print the prefix `warn` method of the commands. This way you can get the length of the prefix such that you can set the desired indent.